config.md

```diff
@@ -57,13 +57,13 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
   * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
   * Solaris: corresponds to "dir" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`type`** (string, OPTIONAL) The filesystem type of the filesystem to be mounted.
-  * Linux: valid *filesystemtype* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
+  * Linux: valid *filesystem type* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
```

Comment:
> And I don't think it's so necessary to use parameter filesystemtype of mount for explanation…

I was thinking that the current entry (from the ancient 77d44b10) was really indented to be `filesystemtype`, since we don't have a precedence for italicizing English words.  If we're moving away from mount(2)'s `filesystemtype` token, I think we should completely switch to English and use:

> Linux: valid filesystem types supported by the kernel are listed…

Although I'm not clear on how “valid” and “supported by the kernel” interact.  Maybe that will get sorted in #813.